meta-lmp-base/recipes-security/optee/optee-client/tee-supplicant@.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=TEE Supplicant on %i
```

Comment:
I believe the optee-client project itself is already installing a similar service file, check on the installed folder, we might be able to drop this one as well.

This is a change that was done in 4.4.

Reply:
yeah I noticed that as well - was going to mention.
we will have to patch it though as it is somewhat different

Reply:
ah, there are configs for the udev rules. ok